css/index.css

```diff
@@ -1,3 +1,518 @@
+
+.main-content .middle-img {
+  margin-top: 30px;
+}
+
+.main-content .bottom-content {
+  display: flex;
+  flex-flow: row nowrap;
+  justify-content: space-between;
+  margin-top: 37px;
+}
+
+.main-content .bottom-content .text-content {
+  width: 274px;
+  display: inline-block;
+  margin-right: 12px;
+}
+
+.bottom-content {
+  padding-bottom: 40px;
+  border-bottom: 1px solid black;
+}
+
+/* Contact & Footer CSS */
+
+.contact {
+  padding: 39px 0 0;
+}
+
+.contact p {
+  margin: 20px 0;
+}
+
+footer {
+  margin: 32px 0;
+  text-align: center;
+  font-size: 14px;
+}
+
+/* Main page CSS for screens 800px and below */
+@media (max-width: 800px) {
+
+  .container {
+    width: 92%;
+    margin: 0 auto;
+  }
+
+  header {
+    display: flex;
+    flex-flow: column wrap;
+  }
+
+  header nav {
+    display: flex;
+    flex-flow: row wrap;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+   header nav a {
+     font-weight: bold;
+     font-size: 16px;
+   }
+
+  header a {
+    order: -1;
+    margin-top: 7px;
+    margin-bottom: 30px;
+    align-self: center
+  }
+
+  .cta {
+    display: inline;
+    margin-top: 100px;
+  }
+
+  .cta h1 {
+    margin-top: 42px;
+    margin-left: 30px;
+    font-size: 72px;
+  }
+
+  .cta button {
+    margin-left: 30px;
+  }
+
+  .cta img {
+      display:none;
+  }
+  
+  .main-content {
+    margin-top: 50px;
+  }
+
+  .main-content .top-content .text-content {
+    width: 49%;
+    display: inline-block;
+  }
+
+  .main-content .middle-img {
+    width: 100%;
+  }
+
+  .main-content .bottom-content .text-content {
+    width: 30%;
+  }
+}  
+
+/* Main page CSS for screens 500px and below */
+@media (max-width: 500px) {
+
+  .container {
+    width: 92%;
+    margin: 0 auto;
+  }
+
+  header {
+    display: flex;
+    flex-flow: column wrap;
+  }
+
+  header nav {
+    display: flex;
+    flex-flow: column;
+    justify-content: space-between;
+    width: 100%;
+  }
+
+   header nav a {
+     font-weight: bold;
+     font-size: 20px;
+     color: lightgrey;
+     width: 500px;
+     text-align: center;
+     border-top: 1px solid lightgrey;
+     padding: 20px;
+     margin: 0 0;
+   }
+
+   header nav a:last-child {
+    border-bottom: 1px solid lightgrey;
+  }
+
+  header nav a:hover {
+    color: white;
+    background: lightgrey;
+    text-decoration: none;
+  }
+
+  header a {
+    order: -1;
+    align-self: center
+  }
+
+  .cta {
+    display: inline;
+    margin-top: 100px;
+  }
+
+  .cta .cta-text {
+    margin-top: 42px;
+    font-size: 72px;
+    padding: 0 0;
+  }
+
+  .cta h1 {
+    margin-left: 0;
+  }
+
+  .cta button {
+    margin-left: 0;
+  }
+
+  .cta img {
+      display:none;
+  }
+
+  .main-content {
+    padding-top: 0;
+  }
+  
+  .main-content .top-content {
+    display: flex;
+    flex-flow: column nowrap;
+  }
+  
+  .main-content .top-content .text-content {
+    width: 100%;
+    display: inline-block;
+    margin-top: 50px;
+  }
+  
+  .main-content .middle-img {
```

Comment:
This is just nitpicking because I can't find anything else wrong, the image is closer to the About paragraph.